WordPress/Classes/ViewRelated/Activity/BaseActivityListViewController.swift

```diff
@@ -499,7 +499,8 @@ extension BaseActivityListViewController: JetpackBackupStatusViewControllerDeleg
             guard let self = self else {
                 return
             }
-            // TODO: fetch backup status
+
+            self.viewModel.refresh()
```

Comment:
Should we remove the guard and just do:

```suggestion
            self?.viewModel.refresh()
```

Reply:
Done in ed80398f57db02ca9ca2d3862dbb439b436563ca